src/predict/jobs/forecast/daily_forecasts.py

```diff
@@ -0,0 +1,128 @@
+from datetime import datetime
+
+import joblib
+import numpy as np
+import pandas as pd
+
+from config import connect_mongo, configuration
+from transform import get_forecast_data
+
```

Comment:
add import for utility for reading model from cloud storage bucket. @Mnoble-19  from utils import get_trained_model_from_gcs

Reply:
![image](https://user-images.githubusercontent.com/8258229/229299590-788e6ce9-73f1-4ac4-8fd5-ad54600c298a.png)